pydatastructs/linear_data_structures/algorithms.py

```diff
@@ -787,3 +788,56 @@ def longest_common_subsequence(seq1: OneDimensionalArray, seq2: OneDimensionalAr
                     check_mat[i][j] = check_mat[i][j-1]
 
     return OneDimensionalArray(seq1._dtype, check_mat[row][col][-1])
+
+
+def longest_increasing_subsequence(array: Array) -> Integer:
```

Comment:
Try replacing `Integer` with `int`.

Reply:
There are many failures due to this. See, https://travis-ci.com/github/codezonediitj/pydatastructs/builds/217409417